src/Controls/src/Core/IndicatorView.cs

```diff
@@ -107,25 +108,51 @@ public IEnumerable ItemsSource
 			set => SetValue(ItemsSourceProperty, value);
 		}
 
-
 		protected override SizeRequest OnMeasure(double widthConstraint, double heightConstraint)
 		{
 			var baseRequest = base.OnMeasure(widthConstraint, heightConstraint);
 
 			if (IndicatorTemplate != null)
 				return baseRequest;
 
-			var defaultSize = IndicatorSize + DefaultPadding + DefaultPadding + 1;
-			var items = Count;
+			var padding = DefaultPadding;
+#if __IOS__
+			padding += 7;
+#endif
+			var defaultSize = IndicatorSize + padding + padding;
+			var items = GetMaximumVisible();
+
 			var sizeRequest = new SizeRequest(new Size(items * defaultSize, IndicatorSize), new Size(10, 10));
```

Comment:
We can create a constant like `DefaultIndicatorViewSize`.

Reply:
for `new Size(10,10)` ?

Reply:
Done